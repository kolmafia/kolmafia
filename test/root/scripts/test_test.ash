boolean ok;
ok = cli_execute("test cookies");
print("cookies " + to_string(ok));
ok = cli_execute("test itemids saucepan");
print("saucepan " + to_string(ok));
