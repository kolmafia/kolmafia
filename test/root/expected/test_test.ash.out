No server cookies
cookies true
saucepan has 1 itemid: 7
saucepan true
